dist/index.js

```diff
@@ -175,6 +176,33 @@ try {
     console.log("Keys added:");
     child_process.execSync('ssh-add -l', { stdio: 'inherit' });
 
+    child_process.execFileSync('ssh-add', ['-L']).toString().split(/\r?\n/).forEach(function(key) {
+        let parts = key.match(/\bgithub.com[:/](.*)(?:\.git)?\b/);
```

Comment:
This regex ensures this will only work for github.com. A few of the people commenting on #38 and other issues mentioned pulling from other services. #38 explicitly had support for bitbucket, Azure DevOps, and others.

Reply:
Yes, a deliberate and defensive measure to keep this feature scoped to GitHub, for now. 

I don't know how widespread the restriction is to have deploy keys limited to one repo. A quick Google search turns up that at least BitBucket seems [not to have this restriction](https://bitbucket.org/blog/deployment-keys).

Reply:
This is why I just had a separate list of URLs rather than trying to parse them out of certificate comments that are out of my control.